tests/unittests/sources/test_openstack.py

```diff
@@ -85,6 +85,13 @@ def mock_is_resolvable():
         yield
 
 
+def mock_distro():
```

Comment:
Do we think it's cleaner to mock these specific attrs here vs using [tests/unittests/util.py:MockDistro](https://github.com/canonical/cloud-init/blob/main/tests/unittests/util.py#L74). It's tough to boil the ocean in our MockDistro, so I can side with wanting this separation here in the specific mock, but I wanted to confirm your thoughts.

Reply:
Good point. I updated the global mock accordingly and deleted the local one.